fdbserver/Knobs.h

```diff
@@ -554,6 +554,11 @@ class ServerKnobs : public Knobs {
 	                                                 // become the leader.
 	double MAX_DELAY_CC_WORST_FIT_CANDIDACY_SECONDS;
 	double DBINFO_FAILED_DELAY;
+	bool ENABLE_WORKER_HEALTH_MONITOR;
+	double WORKER_HEALTH_MONITOR_INTERVAL;
+	double PEER_LATENCY_DEGRADATION_PERCENTILE;
```

Comment:
maybe add a comment on what each knob mean? It helps people to understand how to tune it without digging into code logic

Reply:
Missed this comment earlier. Done.